pkg/controller/command/didclient/command.go

```diff
@@ -145,7 +145,7 @@ func (c *Command) CreateTrustBlocDID(rw io.Writer, req io.Reader) command.Error
 
 		opts = append(opts, didclient.WithPublicKey(&didclient.PublicKey{
 			ID: v.ID, Type: v.Type, Encoding: v.Encoding,
-			KeyType: v.KeyType, Purpose: v.Purpose, Recovery: v.Recovery, Update: v.Update, Value: value,
+			KeyType: v.KeyType, Purposes: v.Purpose, Recovery: v.Recovery, Update: v.Update, Value: value,
```

Comment:
shouldn't v.Purpose also be v.Purposes?

Reply:
done, I'll have to pay attention now for "purpose" for downstream projects